css/index.css

```diff
@@ -1,3 +1,532 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/
+  v2.0 | 20110126
+  License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, quote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+ margin: 0;
+ padding: 0;
+ border: 0;
+ font-size: 100%;
+ font: inherit;
+ vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+ display: ;
+}
+body {
+ line-height: 1;
+}
+ol, ul {
+ list-style: none;
+}
+quote, q {
+ quotes: none;
+}
+quote:before, quote:after,
+q:before, q:after {
+ content: '';
+ content: none;
+}
+table {
+ border-collapse: collapse;
+ border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+   box-sizing: border-box;
+}
+
+html, body {
+   height: 100%;
+   font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+   font-family: 'Bangers', cursive;
+   letter-spacing: 1px;
+   margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+body {
+ margin: 24px 67px;
+}
+@media(max-width: 800px){
+  body {
+   margin: 24px 20px;
+  }
+}
+
+@media(max-width: 500px){
+  body {
+   margin: 24px 8px;
+  }
+}
+
+img {
+ max-width: 100%;
+
+}
+
+a,
+button {
+ cursor: pointer;
+}
+
+.navigation {
+ height: 3em;
+ margin-bottom: 3em;
+ display: flex;
+ justify-content: space-between;
+ flex-wrap: wrap-reverse;
+}
+
+@media(max-width: 800px) {
+  .navigation {
+    height: 7em;
+    display: flex;
+    justify-content: space-between;
+    flex-direction: column-reverse;
+  }
+	.navigation img{
+		display: none;
+	}
+}
```

Comment:
Why did you choose to make the nav logo image disappear when in tablet width and below (as opposed to the design file)?

It actually looks the same as the design file in tablet view when `display: none;` is taken out (though a different issue arises in mobile view).